tests/test_converter.py

```diff
@@ -2206,3 +2207,79 @@ def test_convert_with_invalid_operator(self):
         match = "The operator 123 is not a valid Qiskit SparsePauliOp."
         with pytest.raises(ValueError, match=match):
             load_pauli_op(123)
+
+
+# pylint:disable = import-outside-toplevel, too-few-public-methods
+class TestLoadNoiseModel:
+    """Tests for :func:`load_noise_models()` function."""
+
+    qiksit = pytest.importorskip("qiskit", "1.0.0")
+
+    @staticmethod
+    def _kraus_to_choi(krau_mats, optimize=False) -> np.ndarray:
+        r"""Transforms Kraus representation of a channel to its Choi representation."""
+        kraus_vecs = np.array([kraus.ravel(order="F") for kraus in krau_mats])
+        return np.einsum("ij,ik->jk", kraus_vecs, kraus_vecs.conj(), optimize=optimize)
+
+    def test_build_noise_model(self):
+        """Tests that _build_noise_model_map constructs correct model map for a noise model"""
```

Comment:
```suggestion
        """Tests that _build_noise_model_map constructs correct model map for a Qiskit noise model"""
```

Reply:
After reading the function -- are we not building a noise model here instead of a noise model map? Should update the docstring in general.